### PR TITLE
script for automatic jormungandr updating

### PR DESCRIPTION
**Pull request by manveru**

### CODE REVIEW

**Comment by rvl** [comment]
That is cool² @manveru.

File: jormungandr-lib/default.nix [hunk 1]
```diff
@@ -11,101 +11,7 @@ let
...
-      sha256 = "0msaj4hl93h0q5ay3w1dmkjmvw0lvwmfpd9hcxc17ldf6jq4f85n";
-      cargoSha256 = "14n96hyb75wycqpzv1sv1qlxqykligr3zyvaa4a804hhk3grpjn7";
-    };
-
-    v0_8_0-rc10 = {
-      version = "0.8.0-rc10";
-      sha256 = "0r8990fqwi8jihjdzvknyif3k7mmg21rs6jbaqmn12r21irlv2zw";
-      cargoSha256 = "0vkn8a84f2dnr7p3jlif4lmrnz4w8y1dy0908kxvacvmdi72vfd5";
-    };
+    release = calculatedVersions.v0_7_0;
```
Comment (jbgi) [inline jormungandr-lib/default.nix]: should this be last stable release? (`v0_7_5`)?

**Pull request merged**